app/backend/src/couchers/utils.py

```diff
@@ -96,6 +97,16 @@ def get_coordinates(geom):
     return (shp.y, shp.x)
 
 
+def to_timezone_lng_lat(lng, lat):
+    """
+    Returns the timezone name corresponding the (lng, lat) input
+    The timezone names are defined in this timezone database:
+    https://www.iana.org/time-zones
+    """
+    tf = TimezoneFinder()
```

Comment:
Does this reload all of TF every time this function is called? Do we need to create a cached `TimezoneFinder` object and call that each time we want to do the actual computation?

Does this function throw an exception on failure? When can it fail?

Reply:
I can initialize with `TimezoneFinder(in_memory=True)` to [cache the relevant data](https://timezonefinder.readthedocs.io/en/latest/1_usage.html#initialisation)